src/CreatorToken.sol

```diff
@@ -46,35 +65,61 @@ contract CreatorToken is ERC721Royalty {
     uint256 creatorFee,
     uint256 adminFee
   );
+
+  /// @dev Event emitted when a token is sold.
   event Sold(
     address indexed seller,
     uint256 indexed tokenId,
     uint256 salePrice,
     uint256 creatorFee,
     uint256 adminFee
   );
+
+  /// @dev Event emitted when the contract pause state is toggled.
   event ToggledPause(bool oldPauseState, bool newPauseState, address caller);
+
+  /// @dev Event emitted when the creator is updated.
   event CreatorUpdated(address oldCreator, address newCreator);
+
+  /// @dev Event emitted when the admin is updated.
   event AdminUpdated(address oldAdmin, address newAdmin);
+
+  /// @dev Event emitted when the tokenURI is updated.
   event TokenURIUpdated(string oldTokenURI, string newTokenURI);
 
+  /// @dev Ensures that the given address is not the zero address.
```

Comment:
Should also be `@notice`

Reply:
updated https://github.com/showtime-xyz/creator-tokens/pull/46/commits/3c46721e1c9a0899f54630e2b73028cd5bef159c